stable/xray/templates/xray-indexer-statefulset.yaml

```diff
@@ -202,6 +208,9 @@ spec:
 {{ toYaml . | indent 8 }}
     {{- end }}
       volumes:
+{{- if .Values.server.customVolumes }}
```

Comment:
This should be indexer.customVolumes

Reply:
Resolved!

Reply:
This is still server.customValues... it should be indexer.customValues

Reply:
Sorry @evannorstrand, but the `if` statement is still using `server.customVolumes` instead of `indexer.customVolumes`.... Can you please change that? This is actually why I like using the `with` statement, because it helps avoid these sort of stuff :)

Reply:
@evannorstrand please see this if statement.... thanks for fixing the README but this part is still using an if statement with server.customVolumes.
This should be indexer.customVolumes